internal/monitors/elasticsearch/client/metricnames.go

```diff
@@ -35,44 +35,44 @@ const (
 	// Flush stats
 	indicesFlushTotal     = "elasticsearch.indices.flush.total"
 	indicesFlushPeriodic  = "elasticsearch.indices.flush.periodic"
-	indicesFlushTotalTime = "elasticsearch.indices.flush.total-time"
+	IndicesFlushTotalTime = "elasticsearch.indices.flush.total-time"
```

Comment:
Remove all these where possible and use the generated constants from genmetadata.go. If there are metrics missing from metadata add them.

Reply:
Is this possible?

Reply:
No, because the metadata constants are not exported so can't use them in package client.